src/Propulsion.Cosmos/PropulsionCosmosPrometheus.fs

```diff
@@ -4,111 +4,145 @@ namespace Propulsion.CosmosStore.Prometheus
 namespace Propulsion.Cosmos.Prometheus
 #endif
 
+/// This file implements a Serilog Sink `LogSink` that publishes metric values to Prometheus.
```

Comment:
could it go to L1 ? (normally there would be a `module` and no ugly ifdefs)

Reply:
Moved